### DISCUSSION TITLE
QA model

### DISCUSSION THREAD

**Discussion by Marwa199527** [aub-mind/arabert#156] [Q&A]
Hi @WissamAntoun;
An hour ago I was running your code and there were no problems whatsoever, and results were printed out BUT when I run the code again without changing absolutely anything this error was printed.

![Capture](https://user-images.githubusercontent.com/68481906/156040820-a6b783a7-3d67-42fa-b660-c002f54682f1.PNG)

Any help is appreciated.

**Comment by WissamAntoun** [comment]
hmm very weird, i think something must have changed in the transformers package. maybe you are using an old version of the transformers library with the most updated run_qa.py file

**Reply by Marwa199527** [reply]
@WissamAntoun I did not change anything for sure and I also re-run your code even without changing the data, the same problem was printed out, I really want this to work, any ideas please share it!
